5.x/crud-how-to.md

```diff
@@ -210,31 +210,35 @@ Say you want to show two selects:
 1. In your CrudController you would do:
 
 ```php
-$this->crud->addField([    // SELECT2
+// select2
+$this->crud->addField([
     'label'         => 'Category',
     'type'          => 'select',
     'name'          => 'category',
     'entity'        => 'category',
     'attribute'     => 'name',
 ]);
 
-$this->crud->addField([ // select2_from_ajax: 1-n relationship
-    'label'                => "Article", // Table column heading
-    'type'                 => 'select2_from_ajax_multiple',
-    'name'                 => 'articles', // the column that contains the ID of that connected entity;
-    'entity'               => 'article', // the method that defines the relationship in your Model
-    'attribute'            => 'title', // foreign key attribute that is shown to user
-    'data_source'          => url('api/article'), // url to controller search function (with /{id} should return model)
-    'placeholder'          => 'Select an article', // placeholder for the select
+// select2_from_ajax: 1-n relationship
+$this->crud->addField([
+    'label'                   => "Article", // Table column heading
+    'type'                    => 'select2_from_ajax_multiple',
+    'name'                    => 'articles', // the column that contains the ID of that connected entity;
+    'entity'                  => 'article', // the method that defines the relationship in your Model
+    'attribute'               => 'title', // foreign key attribute that is shown to user
+    'data_source'             => url('api/article'), // url to controller search function (with /{id} should return model)
+    'placeholder'             => 'Select an article', // placeholder for the select
     'include_all_form_fields' => true, //sends the other form fields along with the request so it can be filtered.
-    'minimum_input_length' => 0, // minimum characters to type before querying results
-    'dependencies'         => ['category'], // when a dependency changes, this select2 is reset to null
-    // 'method'                    => 'GET', // optional - HTTP method to use for the AJAX call (GET, POST)
+    'minimum_input_length'    => 0, // minimum characters to type before querying results
+    'dependencies'            => ['category'], // when a dependency changes, this select2 is reset to null
+    // 'method'               => 'POST', // optional - HTTP method to use for the AJAX call (GET, POST)
 ]);
 ```
 
 **DIFFERENT HERE**: ```minimum_input_length```,  ```dependencies``` and ```include_all_form_fields```.
 
+Note: if you are going to use `include_all_form_fields` we recommend you to set the method to `POST`, and to properly setup that in your routes. Since all the fields in the form are going to be sent in the request, `POST` support more data.
+
 2. That second select points to routes that need to be registered:
```

Comment:
Shouldn't we also use POST on the routes below, if we're to recommend using POST?

Reply:
Makes total sense of course 👌

Reply:
Done @tabacitu 👌
I also switched stuff in demo, from GET to POST; https://github.com/Laravel-Backpack/demo/pull/412.
Not 100% sure, but, I think we should right?